app/models/operatingsystem.rb

```diff
@@ -284,4 +285,4 @@ def reject_empty_provisioning_template(attributes)
     attributes.merge!({:_destroy => 1}) if template_exists && provisioning_template_id_empty
     (!template_exists && provisioning_template_id_empty)
   end
-end
+end
```

Comment:
Nitpick, please keep the \n here, your editor might be configurable to keep EOLs.